test/e2e/tests/swap-eth.spec.js

```diff
@@ -0,0 +1,93 @@
+const { strict: assert } = require('assert');
+const { withFixtures, largeDelayMs } = require('../helpers');
+
+describe('Swap Eth for another Token', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it('Completes a Swap between Eth and Dai', async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        await driver.clickElement(
+          '.wallet-overview__buttons .icon-button:nth-child(3)',
+        );
+        await driver.clickElement(
+          '[class*="dropdown-search-list"] + div[class*="MuiFormControl-root MuiTextField-root"]',
+        );
+        await driver.fill('input[placeholder*="0"]', '2');
+        await driver.clickElement(
+          '[class="dropdown-search-list__closed-primary-label dropdown-search-list__select-default"]',
+        );
+        await driver.fill(
+          '[placeholder="Search for a token"]',
+          'Dai stablecoin',
+        );
+        await driver.waitForSelector(
+          '[class="searchable-item-list__primary-label"]',
+        );
+        await driver.clickElement(
+          '[class="searchable-item-list__primary-label"]',
+        );
+        await driver.clickElement('[data-testid="page-container-footer-next"]');
+        await driver.waitForSelector('[class*="box--align-items-center"]');
+        const estimatedEth = await driver.waitForSelector({
+          css: '[class*="box--align-items-center"]',
+          text: 'Estimated gas fee',
+        });
+        assert.equal(await estimatedEth.getText(), 'Estimated gas fee');
+        await driver.waitForSelector(
+          '[class="exchange-rate-display main-quote-summary__exchange-rate-display"]',
+        );
+        await driver.waitForSelector(
+          '[class="fee-card__info-tooltip-container"]',
+        );
+        await driver.waitForSelector(
+          '[data-testid="page-container-footer-next"]',
+        );
+        await driver.delay(largeDelayMs * 3);
```

Comment:
Instead of adding delays, can we wait until the `data-testid="page-container-footer-next` element is visible? That way we can speed up our tests a bit.

Reply:
Hey Dan I tried myself to not  use the delay before and to wait for the data-testid="page-container-footer-next.  I tried having it wait until multiple elements are displayed. The problem is the test moves too fast without the delay, and it actually skips the Transactions complete display without a delay